### ISSUE TITLE
Error when deploy with capistrano is finished

### ISSUE THREAD

**Issue by KieranP** [DataDog/dogapi-rb#148]
```
triggering exit callbacks
* 2018-04-09 14:37:19 executing `datadog:submit'
Could not submit to Datadog: #<NoMethodError: undefined method `round' for nil:NilClass>
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:112:in `block in report'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:100:in `map'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:100:in `report'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:29:in `submit'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog/v2.rb:61:in `block (3 levels) in <module:Datadog>'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/execution.rb:138:in `instance_eval'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/execution.rb:138:in `invoke_task_directly'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/callbacks.rb:25:in `invoke_task_directly_with_callbacks'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/execution.rb:89:in `execute_task'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog/v2.rb:27:in `block in find_and_execute_task'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:70:in `block in record_task'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/2.1.0/benchmark.rb:279:in `measure'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog.rb:69:in `record_task'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/dogapi-1.29.0/lib/capistrano/datadog/v2.rb:25:in `find_and_execute_task'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/callback.rb:38:in `call'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/callbacks.rb:141:in `block in trigger'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/callbacks.rb:141:in `each'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/configuration/callbacks.rb:141:in `trigger'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/cli/execute.rb:35:in `execute!'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/lib/capistrano/cli/execute.rb:14:in `execute'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/capistrano-2.15.9/bin/cap:4:in `<top (required)>'
/Users/Work/.rbenv/versions/2.1.10/bin/cap:23:in `load'
/Users/Work/.rbenv/versions/2.1.10/bin/cap:23:in `<top (required)>'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `load'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `kernel_load'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:28:in `run'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli.rb:424:in `exec'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli.rb:27:in `dispatch'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/cli.rb:18:in `start'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/exe/bundle:30:in `block in <top (required)>'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
/Users/Work/.rbenv/versions/2.1.10/lib/ruby/gems/2.1.0/gems/bundler-1.16.1/exe/bundle:22:in `<top (required)>'
/Users/Work/.rbenv/versions/2.1.10/bin/bundle:23:in `load'
/Users/Work/.rbenv/versions/2.1.10/bin/bundle:23:in `<main>'
```

https://github.com/DataDog/dogapi-rb/blob/master/lib/capistrano/datadog.rb#L112

**Comment by TheDevster** [comment]
I am having this same problem, is there a solution?

**Comment by adamgamble** [comment]
I'm also having this issue.

**Comment by sorefull** [comment]
@TheDevster @adamgamble 
Any progress?

**Comment by rmoriz** [comment]
see my PR in #158 and give it a try.

**Comment by masci** [comment]
This was closed by merging #158 feel free to reopen if the fix doesn't work
